gcs/src/components/config/motorTest.jsx

```diff
@@ -13,14 +15,69 @@ import tailwindConfig from '../../../tailwind.config'
 const tailwindColors = resolveConfig(tailwindConfig).theme.colors
 
 // Custom helper function
+import { useLocalStorage } from '@mantine/hooks'
+import { FRAME_CLASS_MAP, LETTERS } from '../../helpers/mavlinkConstants'
 import { socket } from '../../helpers/socket'
 
-export default function Motortestpanel({
-  selectedThrottle,
-  selectedDuration,
-  setSelectedThrottle,
-  setSelectedDuration,
-}) {
+export default function Motortestpanel() {
```

Comment:
Change the name of this component to `MotorTestPanel`, components should be PascalCase